cmd/metal-api/internal/service/machine-service.go

```diff
@@ -328,6 +328,16 @@ func (r machineResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", v1.MachineRecentProvisioningEvents{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.POST("/events").
```

Comment:
Up to this point we have no plural in our API namings, I think it's more consistent to use `/event`.

Reply:
Done